### ISSUE TITLE
ERROR: Couldn't find env file: secrets_sql.env

### ISSUE THREAD

**Issue by asimzeeshan** [jjethwa/icinga2#163]
When I clone and run with docker-compose, I get this error

    root@surveil:/home/icinga2# docker-compose up -d --build
    ERROR: Couldn't find env file: /home/icinga2/secrets_sql.env

**Comment by bebehei** [comment]
Oooops, I totally forgot to add documentation for this file.
All you have to do is setting an env var in it and filling the password:
```
cat > secrets_sql.env <<-END
MYSQL_ROOT_PASSWORD=<somepassword>
END
```

**Comment by asimzeeshan** [comment]
Thanks so much, it worked

**Comment by bebehei** [comment]
Fixed in d4550bcc8f609df62569f78701788276fcb5df29
